tests/rag_answers/test_evaluate.py

```diff
@@ -75,17 +92,20 @@ def test_per_input_metric_averages(self, mock_evaluation_results):
             ("mean", "faithfulness"),
             ("std", "bias"),
             ("std", "faithfulness"),
+            ("n_datapoints", "bias"),
+            ("n_datapoints", "faithfulness"),
         ]
-        assert list(metric_averages[("name", "")]) == ["Test1", "Test2"]
+        assert list(metric_averages[("name", "")]) == ["Test1", "Test2", "Test3"]
         assert list(metric_averages[("input", "")]) == [
             "Is Vat a tax?",
             "What is capital of France?",
+            "What error can occur?",
         ]
```

Comment:
Shall we add an assertion that the datapoints have been counted correctly as a way of getting quite a strong regression test.

For what it's worth I think it might be simpler to add an error to one of the existing two EvaluationResults than maintaining three as we're not really making use of the distinction between the existing two.